stats/stats.go

```diff
@@ -422,6 +422,27 @@ func (h *Histogram) copyHDataFrom(src *Histogram) {
 	}
 }
 
+// Merge two different histogram with different scale parameters
+// Highest offset and divider value will be selected on new Histogram to maximize scope
+func Merge(h1 *Histogram, h2 *Histogram) *Histogram {
+	var divider float64
+	var offset float64
+	if h2.Divider > h1.Divider {
+		divider = h2.Divider
+	} else {
+		divider = h1.Divider
+	}
+	if h2.Offset > h1.Offset {
```

Comment:
I would think you want the smallest offset to not miss data ?

Reply:
Yes, you are right. I didnt think like that.